### ISSUE TITLE
How to map custom fields having epic link pointing to the epic issue type

### ISSUE THREAD

**Issue by smehto** [solidify/jira-azuredevops-migrator#360]
Hello Team,
My Issue: When i export Epic, all the "subtask" of epic exported but "Issues in Epic" not exported. is there any configuration to add "Issue in epic "? (refer screen shot)
"Issues in Epic" is similar to Sub-tasks in Issue, and that's it.
![image](https://user-images.githubusercontent.com/80322213/118439482-b5ee2480-b703-11eb-8e63-a2658373728c.png)

Q1. How to migrate custom fields having epic link and when i click on that link it will redirect to epic jira?
JIRA XML

![image](https://user-images.githubusercontent.com/80322213/118439592-e8981d00-b703-11eb-9b0b-5965d565bc6d.png)
image

Q2. Can I map custom field in jira with link type in azure?

We have a below kind of scenario, please help here with how to migrate?
Those issues with an Epic Link pointing to the name of an Epic will be included in the "Issues in Epic" section of that Epic. For example:
Issue-1 is of Epic issue type and has the name Epic-1
Issue-2 is of standard issue type and its Epic Link field has the value Epic-1
then Issue-2 will be included in the "Issues in Epic" section of Issue-1
Basically, Epic Link of a standard issue is linked to Epic Name of an Epic.

AB#5024

**Comment by Alexander-Hjelm** [comment]
Try using the Epic Link Field in the config. You can find more information in the FAQ: https://github.com/solidify/jira-azuredevops-migrator/blob/master/docs/config.md

Closed due to inactivity. Feel free to open again if this is still an issue
